# variables of the message

string robot
string last_event
float32[] param 
string[] possible_events
string[] disabled_events
string event_time
